base_comment_template/migrations/14.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+field_renames = [
+    ("base.comment.template", "base_comment_template", "priority", "sequence"),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Not tested
```

Comment:
This should be executed only if coming from 14.0.1.0.0 to 14.0.2.0.0, so change the folder name, and we need to check if we are coming from 13.0.x or from 14.0.1.0.0, so look for a data condition that is only in 14.0.1.0.0.

It should be basically the same as what we have done from 13.0.1.0.0 to 13.0.2.0.0, that is tested, so remove this not elegant comment and make sure it works.